webui/src/app/stores/graphs-store.service.ts

```diff
@@ -3,8 +3,9 @@ import { createFeatureSelector, createSelector, Selector, Store } from '@ngrx/st
 
 import { ActionWithPayloadAndPromise } from './action-with-payload';
 import { ApplicationBean, ApplicationPagesBean, DeploymentBean, DomainBean, DomainPagesBean } from '../models/commons/applications-bean';
-import { Subject } from 'rxjs/Subject';
+import { Subject } from 'rxjs';
 import { GraphBean } from '../models/graph/graph-bean';
+import { Observable } from 'rxjs/internal/Observable';
```

Comment:
Sounds very weird to rely on someone else' internals! Auto-import typo for `rxjs/Observable`?